website/package.json

```diff
@@ -158,7 +158,7 @@
     "@11ty/eleventy-img": "^3.1.8",
     "@11ty/eleventy-navigation": "^0.3.5",
     "@11ty/eleventy-plugin-syntaxhighlight": "^5.0.0",
-    "@custom-elements-manifest/analyzer": "^0.9.3",
+    "@custom-elements-manifest/analyzer": "^0.9.2",
```

Comment:
is this deliberate or a merging mistake?

Reply:
Good one! Merging mistake.